components/molecules/OnboardingButton/onboarding-button.tsx

```diff
@@ -5,7 +5,7 @@ import ProgressPie from "components/atoms/ProgressPie/progress-pie";
 
 const OnboardingButton: React.FC = () => {
   return (
-    <button className="flex items-center gap-2 border border-light-orange-8 py-1 px-1 pr-3 rounded-lg">
+    <button className="hidden items-center gap-2 border border-light-orange-8 py-1 px-1 pr-3 rounded-lg md:flex">
```

Comment:
In my opinion we should show the onboarding button on the small screens instead of just hiding it, what we need is to fix the style.